.github/workflows/test.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     - name: Docs
       if: startsWith(matrix.os, 'ubuntu') && matrix.python-version == 3.9
       run: |
-        python3 -m pip install sphinx-issues sphinx-removed-in sphinx-rtd-theme
+        python3 -m pip install sphinx_copybutton sphinx-issues sphinx-removed-in sphinx-rtd-theme sphinxext-opengraph
```

Comment:
```suggestion
        python3 -m pip install sphinx-copybutton sphinx-issues sphinx-removed-in sphinx-rtd-theme sphinxext-opengraph
```

Reply:
Apparently it doesn't actually cause a problem for pip - https://github.com/python-pillow/Pillow/pull/5498/checks?check_run_id=2589756480#step:13:8 - but you know.